catalog/core/catalog-core-api/src/main/java/ddf/catalog/Constants.java

```diff
@@ -150,4 +150,5 @@
   public static final String SUGGESTION_DICT_KEY = "suggestion-dictionary";
 
   public static final String SUGGESTION_RESULT_KEY = "suggestion-result";
+  public static final String ADDITIONAL_SORTS_BYS = "additional.sorts.bys";
```

Comment:
Seems like this needs a newline to be consistent with the rest of the constants in this class.